src/sensor/settings.py

```diff
@@ -328,6 +345,9 @@
             "PORT": "5432",
         }
     }
+    DEVICE_MODEL = env("DEVICE_MODEL", default="RSA507A")
+    SIGAN_MODULE = env.str("SIGAN_MDOULE", default="scos_tekrsa.hardware.tekrsa_sigan")
+    SIGAN_CLASS = env.str("SIGAN_CLASS", default="TekRSASigan")
```

Comment:
I'm not sure about these defaults. If using another type of signal analyzer, and failing to set the correct environment variables, this would create a confusing situation where Tektronix-specific modules try to load. Maybe these defaults should not work at all, forcing the user to correctly provide the environment variables?

Reply:
Updated in https://github.com/NTIA/scos-sensor/commit/4dcfd4faeeb23a1587c1294161f549c3280cc55d to default to none and added check to ensure SIGAN_MODULE and SIGAN_CLASS are set.